styles/app.scss

```diff
@@ -4,3 +4,222 @@
 // $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 // @import 'font-awesome';
 @import 'box-sizing';
+@import 'queries';
+
+nav {
+  width: 100%;
+  padding: 1rem;
+  background: black;
+  display: flex;
+  justify-content: center;
+}
+
+.logo {
+  width: 100%;
+  height: 3rem;
+}
+
+.nav-words {
+  display: flex;
+  color: white;
+  padding: 0 1rem;
+  font-size: 1rem;
+}
+
+.nashville {
+  color: inherit;
+  text-decoration: none;
+}
+
+.nashville:hover {
+  transform: scale(1.2);
+  color: #17f112;
+
+}
+
+.top-o-page {
+  text-align: center;
+  padding: 3rem;
+  margin: 0;
+  font-size: 2rem;
+  text-transform: uppercase;
+  background: #41d0e1;
+  font-family: Verdana;
+}
+
+h1 {
+  font-size: 3rem;
+  padding: 2rem 0;
+  margin: 0;
+
+}
+
+h6 {
+  color: #fff;
+  margin: 0;
+  padding-bottom: 2rem;
+}
+
+.projects {
+  text-align: center;
+  font-size: 2.5rem;
+  padding: 2rem;
+  margin: 0;
+}
+
+.bigger-hero {
+  padding: 0 1rem;
+}
+
+.hero {
+  max-width: 65rem;
+  margin: auto;
+}
+
+.project-row {
+  display: flex;
+  width: 100%;
+  font-size: 1.4rem;
+}
+
+.project-row--reverse {
+  display: flex;
+  flex-direction: row-reverse;
+  width: 100%;
+  font-size: 1.4rem;
+}
+.project-text {
+  margin: 1rem;
+}
+.project-frame {
+  margin: 1rem;
+}
+.project-img {
+  width: 100%;
+}
+
+h3 {
+  font-size: 3.3rem;
+  margin: 0;
+}
+
+.footer {
+  display: flex;
+  justify-content: center;
+  background: black;
+  margin-top: 4rem;
+  width: 100%;
+}
+
+.icons {
+  color: #fff;
+  font-size: 2rem;
+  width: 60%;
```

Comment:
Instead of width 60% use margin on the icon links and justify-content: center and you should be able to make the footer work better at different sizes!